Dalamud/DiscordBot/DiscordBotManager.cs

```diff
@@ -315,12 +348,187 @@ public async Task ProcessCfPreferredRoleChange(string rouletteName, string prevR
                         }
                     }
 
-                    await channels.ElementAt(chatTypeIndex).SendMessageAsync($"**[{chatTypeDetail.Slug}]{name}**: {rawMessage}");
+                    string messageName = name;
+                    if (chatTypeDetail.Slug.Equals("fc"))
```

Comment:
i am hopping aboard the review train: ``Equals("fc")`` -> ``== "fc"``?

Reply:
...

Reply:
\uE05A*